lib/game_factory.rb

```diff
@@ -0,0 +1,23 @@
+class GameFactory
+  attr_reader :games
+
+  def initialize
+    @games = []
```

Comment:
I like the way you're creating a collection here. I'll mimic this, it's more dynamic than how I was copying the class lesson (magical_pets) which worked but wasn't accumulating the objects in this fashion